app/buck2_execute/src/re/client.rs

```diff
@@ -1141,6 +1143,9 @@ impl RemoteExecutionClientImpl {
                     ..Default::default()
                 },
             )
+            .inspect_err(|err| {
+                tracing::warn!("write_action_result failed: {err}");
```

Comment:
Does this not appear in https://github.com/facebook/buck2/blob/main/app/buck2_execute_impl/src/executors/caching.rs#L108C25-L108C25 as a warning? It certainly should

Reply:
That line may have run, but I definitely did not see it in the terminal. Thanks for the pointer, I will have a look at why.